src/backup-all-indexes.sh

```diff
@@ -67,6 +95,42 @@ backup_index ()
   curl -w "\n" -sS -XDELETE ${INDEX_URL}
 }
 
+backup_index ()
+{
+  : ${1:?"Error: expected index name passed as parameter"}
+  local INDEX_NAME=$1
+  local SNAPSHOT_URL=${REPOSITORY_URL}/${INDEX_NAME}
+  local INDEX_URL=${DATABASE_URL}/${INDEX_NAME}
+
+  grep -q SUCCESS <(curl -sS ${SNAPSHOT_URL})
+
+  if [ $? -ne 0 ]; then
```

Comment:
There's some major code duplication going on over here with `archive_index` - this would need to be fixed.